libsrc/common/ntohl.s

```diff
@@ -0,0 +1,34 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2023-09-06
+;
+; int __fastcall__ ntohl (long val);
+;
+
+.export     _ntohl, _htonl
+.import     popa
+.importzp   tmp1, tmp2, sreg
+
+_htonl := _ntohl
+.code
```

Comment:
The predominant style is to leverage the fact that the assembler defaults to `.code`.